M2/Macaulay2/packages/KGAlgebras.m2

```diff
@@ -3,132 +3,115 @@ newPackage(
     Version => "0.5",
     Date => "February 19th, 2025",
     Authors => {
-	{Name => "Daniel Koizumi", Email => "daniel.koizumi@utexas.edu"},
+    {Name => "Daniel Koizumi", Email => "daniel.koizumi@utexas.edu"},
     {Name => "Moses Samuelson-Lynn", Email => "m.samuelsonlynn@utah.edu"},
     {Name => "Toshi Mowery", Email => "tosmow@gmail.com"}
     },
-    Headline => "A package for KG-algebras",
-    Keywords => {"KG-algebra"},
-    DebuggingMode => false,
-    Reload=>true,
+    Headline => "A package for group algebras",
+    Keywords => {"Group Theory"},
+    DebuggingMode => true,
```

Comment:
`DebuggingMode` should either be `false` or (even better) removed, since `false` is the default.